css-text-3/Overview.bs

```diff
@@ -824,7 +824,7 @@ Order of Operations</h4>
   Name: white-space
   Value: normal | pre | nowrap | pre-wrap | break-spaces | pre-line
   Initial: normal
-  Applies to: <a>inline boxes</a>
+  Applies to: text
```

Comment:
The text below says

> whether and how <a>white space</a> inside the element is collapsed

Saying "element" seems strange if the property applies to text.

Reply:
Deleted “inside the element”.